manage.py

```diff
@@ -10,7 +10,7 @@ def main():
     """Run administrative tasks."""
     dotenv.load_dotenv()
 
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ap.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "ap.settings.local")
```

Comment:
Why are we setting the default settings to local? What would need to change as part of the deployment process to set it to production settings? To me, this feels like something we should be overriding as part of either the environment or launch json.

Reply:
I've done this because otherwise we are relying on having `DJANGO_SETTINGS_MODULE` set correctly (either in a `.env` file, or in the chart values.yaml) or it will default to the settings that are used in production. This feels riskier than defaulting to local settings, as its easy to imagine a situation where a user pulls down the repo and doesnt have this set before running `manage.py` commands.

It also causes issues when running the tests via a terminal with `pytest` as if `DJANGO_SETTINGS_MODULE` is set in a local .env file to `ap.settings.local`, this will be used rather than what is set in the `pytest.ini` file due to the [order of preference that is used for choosing settings](https://pytest-django.readthedocs.io/en/latest/configuring_django.html#order-of-choosing-settings). This happened to me and it wasnt clear why the `test` settings file was not being used. I know that running the tests via a VS Code debugger config can avoid this, but its nice to quickly be able to run `pytest` sometimes without having to override the local settings or use a flag.

Theres no change required deployment, as we are already setting it as an env var in `chart/values.yaml`